core/src/main/java/org/bitcoinj/core/TransactionConfidence.java

```diff
@@ -337,23 +339,37 @@ public boolean wasBroadcastBy(PeerAddress address) {
     }
 
     /** Return the time the transaction was last announced to us. */
-    public Date getLastBroadcastedAt() {
+    @Nullable
+    public Instant getLastBroadcastedAtInstant() {
         return lastBroadcastedAt;
     }
 
+    /** @deprecated use {@link #getLastBroadcastedAtInstant()} */
+    @Deprecated
+    @Nullable
+    public Date getLastBroadcastedAt() {
+        return lastBroadcastedAt != null ? Date.from(lastBroadcastedAt) : null;
+    }
+
     /** Set the time the transaction was last announced to us. */
-    public void setLastBroadcastedAt(Date lastBroadcastedAt) {
+    public void setLastBroadcastedAt(Instant lastBroadcastedAt) {
         this.lastBroadcastedAt = lastBroadcastedAt;
     }
 
+    /** @deprecated use {@link #setLastBroadcastedAt(Instant)} */
+    @Deprecated
```

Comment:
Was this every set to `null` before?

Reply:
This method is not meant to be called with `null`, if that's what you're asking.